dotcom-rendering/src/components/Picture.tsx

```diff
@@ -12,6 +13,34 @@ import type { Loading } from './CardPicture';
 
 export type Orientation = 'portrait' | 'landscape';
 
+const timelineBulletStyles = css`
+	position: relative;
+	::before {
+		content: '';
+		position: absolute;
+		display: block;
+		width: 12px;
+		height: 12px;
+		border: 1px solid ${palette('--timeline-event-border')};
+		border-radius: 1000px;
```

Comment:
If you’ve set the width and height explicitely, you can set this to 100% or 12px

```suggestion
		border-radius: 100%;
```